cuctf_2019.html

```diff
@@ -11,8 +11,7 @@
       <img src="/public/images/2019/Trophies.png" alt="A line of wooden trophies sits on table next to a CUCTF banner with the subheading sponsored by SecureWorks Adversary Group.">
     </div>
     <div class="col s12 m6">
-      <p class="flow-text">CUCTF 2019 was our first time hosting a Capture the Flag event, and it was a blast! From the array of sponsors to the excited universities to the helpful staff&mdash;the atmosphere was electric.</p>
-    </div>
+<p class="flow-text">CUCTF 2019 was our first time hosting a Capture the Flag event, and it was a blast! From the array of sponsors, excited universities, and helpful staff&mdash;the atmosphere was electric. Below are the winners from last year's competition, we can't wait for CUCTF 2020!</p>    </div>
```

Comment:
```suggestion
      <p class="flow-text">CUCTF 2019 was our first time hosting a capture the flag event, and it was a blast! Between the array of sponsors, excited universities, and helpful staff, the atmosphere was electric. Below are the winners from last year's competition. We can't wait for CUCTF 2020!</p>
    </div>
```

Reply:
I like Foster's changes. The only nitpick I would change is "can't" to "cannot" because contractions should not be used in business-y writing. But again, no big deal if you leave it as is.

Reply:
This is a much more informal sentence and it's semi-intended to feel that way instead of feel like dry emails you receive at work or something. I think there's no harm in keeping the contraction and it just kinda sounds weird saying that sentence with "cannot" instead of "can't".